plugin/src/withSentry.ts

```diff
@@ -55,6 +55,17 @@ export function getSentryProperties(config: ExpoConfig): string | null {
     return '';
   }
 
+  if (sentryHook.config?.authToken) {
+    WarningAggregator.addWarningAndroid(
+      'sentry-expo',
+      'Sentry `authToken` found in app.json. Avoid committing this value to your repository, configure it through `SENTRY_AUTH_TOKEN` environment variable instead. See: https://docs.expo.dev/guides/using-sentry/#app-configuration'
+    );
+    WarningAggregator.addWarningIOS(
+      'sentry-expo',
+      'Sentry `authToken` found in app.json. Avoid committing this value to your repository, configure it through `SENTRY_AUTH_TOKEN` environment variable instead. See: https://docs.expo.dev/guides/using-sentry/#app-configuration'
+    );
+  }
```

Comment:
`"authToken"` also needs to be omitted from the `missingProperties` array at line 74 so that it doesn't warn the user that the field is missing.

Reply:
The missing properties doesn't warn users when not defined. The only thing it's doing is adding a comment within the generated `sentry.properties` file to mention that it's falling back to the `SENTRY_AUTH_TOKEN`.

We could modify that comment, but I do think it could be useful when users stumble upon the configuration after running `$ npx expo prebuild`.

Reply:
I've updated the message, and added another comment when using `authToken`. Just to cover users ignoring the warning, and opening the `sentry.properties` file.

Reply:
@byCedric Could you double check that? I'm currently forced to set `authToken` to `false` in the config in order for the WarningAggregator message to disappear.

Reply:
@jamsch, can you try on commit #e1ce56a? that should take care of that.

Reply:
@jamsch, just released this: `6.1.1`. Thanks for your help :)